scheduler/test/cook/test/authorization.clj

```diff
@@ -33,6 +33,8 @@
     (is (true? (auth/open-auth {} "foo" :create test-job)))
     (is (true? (auth/open-auth {} "bar" :read test-job)))
     (is (true? (auth/open-auth {} "baz" :update test-job)))
+    (is (true? (auth/open-auth {} "bazz" :get test-job)))
```

Comment:
Do we need both `:get` and `:read` in all these tests?

Reply:
Honestly, I have no idea what `:read` is... It looks like it's only used on `/queue` and `/running`, so it probably doesn't even make sense to test these verbs on jobs. I think the same applies for most of the other verbs here. But I'd prefer changing that in a separate PR since it's not retry-related (both the `:get` and `:retry` verbs are used on the `/retry` endpoint, which is why I added those).